charts/substra-backend/templates/_helpers.tpl

```diff
@@ -408,3 +399,47 @@ Retrieve AWS environment variable value
 {{- end -}}
 {{- $value -}}
 {{- end -}}
+
+{{/*
+Define database secret name
+*/}}
+{{- define "substra-backend.database.secretName" -}}
+    {{- if .Values.database.auth.existingSecretName -}}
+        {{- .Values.database.auth.existingSecretName }}
+    {{- else -}}
+        {{ include "substra.fullname" . }}-database
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Define redis secret name
+*/}}
+{{- define "substra-backend.redis.secretName" -}}
+    {{- if .Values.redis.auth.existingSecretName -}}
+        {{- .Values.database.auth.existingSecretName }}
+    {{- else -}}
+        {{ include "substra.fullname" . }}-redis
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Define account operators secret name
+*/}}
+{{- define "substra-backend.accountOperator.secretName" -}}
+    {{- if .Values.addAccountOperator.existingSecretName -}}
+        {{- .Values.database.auth.existingSecretName }}
+    {{- else -}}
+        {{ include "substra.fullname" . }}-add-account
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Define object store secret name
+*/}}
+{{- define "substra-backend.objectStore.secretName" -}}
+    {{- if .Values.redis.auth.existingSecretName -}}
+        {{- .Values.database.auth.existingSecretName }}
```

Comment:
🕵️ 
I think the values have not been changed here

Reply:
🤦